src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/StringUtils.java

```diff
@@ -80,6 +80,46 @@ public static int countMatches(CharSequence input, char match) {
                 count());
     }
 
+    /**
+     *
+     * @param text string
+     * @param quote
+     * @return An unquoted string whose outer pair of back-ticks (if any) has been removed
+     */
+    public static String unquoteSingleField(String text, String quote) {
+        if (isQuoted(text, quote)) {
+            return text.substring(1, text.length() - 1);
+        }
+        return text;
+    }
+
+    /**
+     *
+     * @param text
+     * @return A string whose each dot-seperated field has been unquoted from back-ticks (if any)
+     */
+    public static String unquoteFullColumn(String text, String quote) {
+        if (text == null) {
+            return null;
+        }
+        String[] strs = text.split("\\.");
+        for (int i = 0; i < strs.length; i++) {
+            String unquotedSubstr = strs[i];
+            if (isQuoted(unquotedSubstr, quote)) {
+                unquotedSubstr = strs[i].substring(1, strs[i].length() - 1);
```

Comment:
Is it possible to resuse unquoteSingleField?

Reply:
@penghuo We need to figure out if it's necessary to unquote the string text before unqoute it, both in singleField method and fullColumn method. So I think it will add to runtime if reuse the unquoteSingleField method here.

Reply:
@penghuo yes it would be better to reuse the unquoteSingleField method. have changed it, thanks!